### DISCUSSION TITLE
Setting reminders

### DISCUSSION THREAD

**Discussion by mglnsk** [GitJournal/GitJournal#706] [Feature Requests]
Hello,
I would like to add reminders to my notes, as I could in Google Keep or Joplin. Will it be possible to implement in GitJournal?
## Description

**What you'd like to happen:**
I would like to:
1. Select _add reminder_ option
2. Set due date and time 
3. Get notified when it's time

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by roshanjhegde** [comment]
I would like to see that happen as well.

**Comment by vHanda** [comment]
Implementing the logic for reminders can get complex, and I rather delegate that to another app. Maybe one could just create a calendar event which links to the note.

I'm not going to look into this right now, but if you gather more votes, I'll make it a priority.

**Comment by alensiljak** [comment]
This would be one of the things Org Mode has. I.e. Schedule and Deadline properties (in the header) and Event date time tags anywhere in the body.

**Comment by yajo** [comment]
It could be implemented as metadata in the front matter.

**Reply by saileh** [reply]
Having this as metadata would be a powerful feature.
We could change metadata of the note in Obsidian and have it sent to GitJournal.
That would give GitJournal advantage over Obsidian mobile client (which doesn't have this feature) and Google Keep (which some people use only for this feature).
